lib/api/apiUtils/authorization/tagConditionKeys.js

```diff
@@ -0,0 +1,83 @@
+const async = require('async');
+const { auth, s3middleware } = require('arsenal');
+const metadata = require('../../../metadata/wrapper');
+const { decodeVersionId } = require('../object/versioning');
+
+const { parseTagXml } = s3middleware.tagging;
+
+function makeTagQuery(tags) {
+    const tagsArray = Object.keys(tags);
+    tagsArray.map(t => `${t}=${tags[t]}`);
+    return tagsArray.join(',');
```

Comment:
Maaaaybe a bit of a stretch but... 
```suggestion
    return Object.entires(tag)
        .map(([k, v]) => `${k}=${v}`)
        .join(',');
```

Reply:
alternatively
```
    return Object.entries(tag)
        .map(i => i.join('='))
        .join(',');
```

Reply:
ooh that works too